prov/efa/src/rxr/rxr_rma.c

```diff
@@ -631,6 +631,14 @@ ssize_t rxr_rma_inject_write(struct fid_ep *ep, const void *buf, size_t len,
 	struct fi_msg_rma msg;
 	struct iovec iov;
 	struct fi_rma_iov rma_iov;
+	struct rxr_ep *rxr_ep;
+	struct rdm_peer *peer;
+
+	rxr_ep = container_of(ep, struct rxr_ep, util_ep.ep_fid.fid);
+	peer = rxr_ep_get_peer(rxr_ep, dest_addr);
+	if (peer->is_local) {
```

Comment:
nitpick: the braces are not necessary here. same for the other place.

Reply:
Fixed